src/python/pants/backend/codegen/tasks/simple_codegen_task.py

```diff
@@ -341,11 +343,18 @@ def name(cls):
       return 'isolated'
 
     def execute_codegen(self, targets):
+      start = time.time()
       ordered = [target for target in reversed(sort_targets(targets)) if target in targets]
+      sys.stdout.write('.')
       for target in ordered:
         # TODO(gm): add a test-case to ensure this is correctly eliminating stale generated code.
         safe_rmtree(self._task.codegen_workdir(target))
+        sys.stdout.write('.')
+        sys.stdout.flush()
         self._task.execute_codegen([target])
+      whole_time = time.time() - start
+      sys.stdout.write('\tDone.')
+      sys.stdout.write("\t Time: {}".format(whole_time))
```

Comment:
You don't need to print out a timetamp, there is lots of timing output for that.